apps/stats/stats-1.py

```diff
@@ -91,6 +91,17 @@ def __init__( self ) :
 					check = IECore.FileNameParameter.CheckType.MustExist,
 				),
 
+				IECore.FileNameParameter(
+					name = "postScript",
```

Comment:
I wonder if `preScript` or `postLoadScript` would be a better name? It could just be me, but `postScript` initially sounded like it would be a script run after stats gathering, rather than before. I guess its as there is the gaffer script, and the python script. I think for people running the `stats` app, the primary action is stats gathering, so they would associate that as the thing "post" is relative to. The help text does disambiguate this, but it confused me in the PR title.

Reply:
Makes sense - I was aware of the ambiguity but couldn't talk myself into running `preScript` after `script`, but `postLoadScript` sorts that all out nicely. I'm following up with Murray on something interesting he's discovered during testing, so I'll push this change when I've resolved that with him.